roles/zitadel/tasks/main.yaml

```
@@ -0,0 +1,22 @@
+---
+
+- name: Add stable Zitadel Helm repository
+  kubernetes.core.helm_repository:
+    name: zitadel
+    repo_url: https://charts.zitadel.com/
+
+- name: Deploy Zitadel
+  kubernetes.core.helm:
+    chart_ref: zitadel/zitadel
+    chart_version: "{{ zitadel_chart_version }}"
```

With this you don't need to previous task:

```suggestion
    chart_version: "{{ zitadel_chart_version }}"
    chart_repo_url: https://charts.zitadel.com/
```